### ISSUE TITLE
Mac - Docker Quickstart Terminal requires bash

### ISSUE THREAD

**Issue by Terry-Weymouth** [docker/toolbox#172]
Starting Docker Quickstart Terminal - by double click gives this error in terminal: 
[~] % /bin/tcsh --login
. '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
[~] % /bin/tcsh --login
Unknown option: `--login'
Usage: tcsh [ -bcdefilmnqstvVxX ] [ argument ... ].
[~] % . '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
.: Permission denied.

Problem is that the app requires that the use's SHELL be bash (not, as in my case, for example, tcsh).

Workaround is to type the following command in CLI:
bash /Applications/Docker/Docker\ Quickstart\ Terminal.app/Contents/Resources/Scripts/start.sh

Maybe should check `which bash` and make sure that bash is being used instead of $SHELL 
since it appears that bash is a requirement

**Comment by jeffdm** [comment]
Thanks for the issue!

**Comment by Terry-Weymouth** [comment]
some additional information: In a bash shell you should get the following:
echo $0 -> bash (but not in a script)
echo $BASH -> /bin/bash (or something like that)
(whereas $BASH most likely is undefined in non-bash shells; and $0 will have some other value in non-bash shells)

**Comment by geradcoles** [comment]
Same thing hit me, default shell on my Mac is `sh`, script fails with a no such file or directory, but running directly under bash all is well.

**Comment by momoking** [comment]
I encountered the same issue. It was working normally a few hours ago and suddenly, starts to complain that it cannot find the start.sh Reinstalling docker from DockerToolbox did not fix the problem.

Running the start.sh manually works: 

`bash /Applications/Docker/Docker\ Quickstart\ Terminal.app/Contents/Resources/Scripts/start.sh`

**Comment by lucasvc** [comment]
Also, [`start.sh`](https://github.com/docker/toolbox/blob/master/osx/mpkg/quickstart.app/Contents/Resources/Scripts/start.sh#L60)  only checks for `/bin/bash` and not [brew-installed bash 4](http://apple.stackexchange.com/a/24635/77314).
So no sourcing of `.bash_profile`. The workaround in this case is to softlink binaries.

**Comment by ThomasChambonDev** [comment]
My contribution to this thread.
On mac OS X 10.9.5, the default shell is zsh, I get this output when launching Docker Quickstart Terminal :

>bash --login '/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh'
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 14: clear: command not found
Creating Machine default...
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 27: rm: command not found
Driver "virtualbox" not found. Do you have the plugin binary accessible in your PATH?
Host does not exist: "default"
Starting machine default...
Host does not exist: "default"
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 37: yes: command not found
Regenerate TLS machine certs?  Warning: this is irreversible. (y/n): EOF
Setting environment variables for machine default...
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 41: clear: command not found
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 43: cat: command not found
Host does not exist: "default"
docker is configured to use the default machine with IP 
For help getting started, check out the docs at https://docs.docker.com

>Host does not exist: "default"
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 63: dscl: command not found
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 63: awk: command not found
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts/start.sh: line 63: head: command not found

**Comment by ThomasChambonDev** [comment]
My bad, actually.
The problem I had was linked to a modification of the ~/.bash_profile file when I was setting up the GOPATH variable for the Go language.
It has nothing to do with docker then.

**Comment by Terry-Weymouth** [comment]
Docker must run in bash. On mac OS X, this means that you need to start the
bash shell. I help my myself remember with this alias in my .cshrc file:
alias docker "echo 'use bash' "


From:  Thomas Chambon <notifications@github.com>
Reply-To:  docker/toolbox
<reply+0009ad3db29a55f77b0fdd1f16b5643c6a9fa89d140f0b7a92cf0000000112811f509
2a169ce0652234c@reply.github.com>
Date:  Thursday, December 10, 2015 at 6:09 AM
To:  docker/toolbox <toolbox@noreply.github.com>
Cc:  Terry Weymouth <weymouth@umich.edu>
Subject:  Re: [toolbox] Mac - Docker Quickstart Terminal requires bash
(#172)

**Comment by ThomasChambonDev** [comment]
Thanks for your reply !
Actually I've just figured out that the issue I had was not linked to
Docker but to the setup of the Go environment.
I had modified the ~/.bash_profile file to set the GOPATH permanently, and
it seemed to have erased the bash commands.

I corrected that and it works fine now.

Thanks again for your time, and sorry for not seeing this earlier.

2015-12-10 15:34 GMT+01:00 Terry Weymouth <notifications@github.com>:
